bedrock/firefox/templates/firefox/products/developer-quantum.html

```diff
@@ -77,7 +77,7 @@ <h3 class="body-title">{{ _('Firefox DevTools') }}</h3>
           {% trans %}
             The new Firefox DevTools are powerful, flexible, and best of all,
             hackable. This includes a best-in-class JavaScript debugger, which
-            can target multiple browsers and is built in React and Ember.
+            can target multiple browsers and is built in React and Redux.
           {% endtrans %}
```

Comment:
As mentioned, this will need to be behind a l10n tag so, you are going to want to do something like:

```
{% if l10n_has_tag('ember_reference_fix') %}
  {% trans %}
  The new Firefox DevTools are powerful, flexible, and best of all, hackable. This includes a best-in-class 
  JavaScript debugger, which can target multiple browsers and is built in React and Ember.
  {% endtrans %}
{% else %}
  {% trans %}
  The new Firefox DevTools are powerful, flexible, and best of all, hackable. This includes a best-in-class 
  JavaScript debugger, which can target multiple browsers and is built in React and Redux.
  {% endtrans %}
{% endif %}
```

Reply:
@schalkneethling I removed the package-lock.json file and added it to .gitignore. I'll add this tag next. Thanks for all your help on this.

Reply:
@schalkneethling I added the l10n tag